L78-python-rich-server-context.md

```diff
@@ -0,0 +1,73 @@
+Title
+----
+* Author(s): Chris St. Pierre
+* Approver: a11r
+* Status: Draft
+* Implemented in: python
+* Last updated: 2021-02-17
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Don't forget to create a thread on the mailing list for this. There's a mandatory 2 week period for submission of comments before the code can be merged.